mobile/src/main/java/org/openhab/habdroid/util/ExtensionFuncs.kt

```diff
@@ -499,9 +499,14 @@ fun Uri.Builder.appendQueryParameter(key: String, value: Boolean): Uri.Builder {
 }
 
 fun ServiceInfo.addToPrefs(context: Context) {
+    if (context.getPrefs().getConfiguredServerIds().isNotEmpty()) {
```

Comment:
I'd prefer doing this at the caller site, since the method name doesnt imply only one service info can ever be added to prefs.

Reply:
Done. I added the check to `MainActivity` only, because it's executed after `IntroActivity`.